jobs/service-fabrik-scheduler/templates/config/settings.yml.erb

```diff
@@ -24,6 +24,7 @@ production:
   release_version: '<%= spec.release.version %>'
   username: '<%= link("broker").p('username') %>'
   password: '<%= link("broker").p('password') %>'
+  enable_service_fabrik_v2: '<%= link("broker").p('enable_service_fabrik_v2') %>'
   skip_ssl_validation: <%= link("broker").p('skip_ssl_validation') %>
   session_store:
     path: <%= "/var/vcap/store/#{broker_name}/session" %>
```

Comment:
Do we need to add etcd properties as well?

Reply:
Also is it needed to add `export SF_EVENTMESH_SETTINGS_PATH=$CONF_DIR/settings.yml` in _jobs/service-fabrik-scheduler/templates/bin/job_properties.sh.erb_

Reply:
Added properties in Scheduler as it is required in `BnRStatusPoller` job now.
Removed `SF_EVENTMESH_SETTINGS_PATH`